config/locales/en.yml

```diff
@@ -1561,8 +1561,7 @@ applyNext:
       cta: Start new application
   summary:
     title: Summary
-    introduction: >
-      <p>Here's a summary of your application. You may complete the sections in any order.</p>
```

Comment:
Removing "Here's a summary of your application" here as it's the only text on this page that specifies this is an application and we are keen to test peoples assumptions around that without leading them. Means it will change on the awards for all form but felt acceptable as it basically repeats the title, it's already clear this is a summary.